focuspoints.lrdevplugin/ExifUtils.lua

```diff
@@ -30,8 +31,7 @@ exiftoolWindows = LrPathUtils.child(exiftoolWindows, "exiftool.exe")
 
 function ExifUtils.getExifCmd(targetPhoto)
   local path = targetPhoto:getRawMetadata("path")
-  local metaDataFile = LrPathUtils.removeExtension(path)
-  metaDataFile = metaDataFile .. "-metadata.txt"
+  local metaDataFile = LrPathUtils.child(LrPathUtils.getStandardFilePath("temp"), LrUUID.generateUUID() .. ".txt")
```

Comment:
if this is to a folder, then it's not deleting the folder afterwards. Also, this doesn't fix any of the concerns such as read/write perms. So how about just 
``` metaDataFile .. LrUUID.generateUUID() .. ".txt"```